### ISSUE TITLE
[EN][Item] Cunning of the Cruel

### ISSUE THREAD

**Issue by Psychobird7** [Atlantiss/BugTracker#9491]
**Description**: DS trinket 

**Current behaviour**: Nothing yet 

**Expected behaviour**: Cunning of the Cruel, all versions: 
- LFR: http://www.wowhead.com/item=77980 ; uses http://www.wowhead.com/spell=109799 and triggers http://www.wowhead.com/spell=109798 
- Normal: http://www.wowhead.com/item=77208 ; uses http://www.wowhead.com/spell=108006 and triggers http://www.wowhead.com/spell=108005 
- Heroic: http://www.wowhead.com/item=78000 ; uses http://www.wowhead.com/spell=109801 and triggers http://www.wowhead.com/spell=109800 

All of the above have a 45% proc chance on damaging spell hit (including DoT ticks), and a 9s ICD, as of 4.3.2. 

Damage: 
- The spell called Shadowbolt Volley triggers a volley of shadowbolts that deal shadow damage to every target within 10 yards of the triggering spell's target (**not the player's current target**), including that target. 
- LFR: 2498 to 3747 (SP mod: 0.277) 
- Normal: 2820 to 4230 (SP mod: 0.313) 
- Heroic: 3183 to 4774 (SP mod: 0.353) 
- The above are all confirmed by Wowhead, Simcraft, SpellWorks and Rawr. 
- This is affected by damage modifiers, and can crit *and miss*. 
- No evidence of interactions with Dragonwrath, so it is to be assumed it does not get duplicated. 

Other sources: 
- Damage and scalings were double-checked against a T13 BiS geared Shadow Priest's SP, and the numbers do align with both the log and the simulated damage. 
- http://worldoflogs.com/reports/rt-gw69wt4aubq3auby/details/33/?s=2199&e=2472 (see Damage by spell) 
- http://www.wowhead.com/item=77208/cunning-of-the-cruel#comments:id=1578264 

**Steps to reproduce the problem**:

1. Don't. 

**Revision**: 5482

**Comment by blinkmaster** [comment]
Also every single missle from the volley should be able to be duplicated

**Comment by Psychobird7** [comment]
Bring some evidence to support that claim. Thank you in advance.

**Comment by blinkmaster** [comment]
https://www.youtube.com/watch?v=y7tIIzF6o8k
2:00

**Comment by Psychobird7** [comment]
The video you linked shows no evidence related to this report. Please stop posting if you're going to troll.

**Comment by blinkmaster** [comment]
2:53 it already proc'd from frost spell. wtf are you talking about.

**Comment by Psychobird7** [comment]
- "Cunning of the Cruel" 
- "Also every single missle from the volley should be able to be duplicated" 
- Then you link *that* 

--> :thinking:

**Comment by PolpoVolante** [comment]
I got the heroic version,
What i noticed:

The scaling does not work: https://imgur.com/NkaOkzA

11K sp and 7.6k sp no difference.
Tooltip damage says it should do 7894 per shadow bolt. (correct)

The internal cooldown is 9 seconds.

Dragonwrath procs entire volley and it should not, it should proc single bolts on the targets they proc off of. Below a comment i found.

"It won't proc another volley, but I believe our warlocks (have 2 with DTR+Cunning HC lol) said it can proc off a single shadowbolt (from the proc)."

source: https://www.mmo-champion.com/threads/1074500-Dragonwrath-Cunning-of-the-Cruel

However I noticed that sometime a first batch of volley gets shot lets say hitting 3 targets but when dragonwrath procs it hits more than 3 targets.
Probably because the mirrored spell that is being cast uses the same script as the trinket and because of that it can affect more targets.
This is a "collateral" bug, If the single procs get fixed this won't happen again.
